### PR TITLE
Poly_comm: benchmark add and sub over Pallas/Vesta

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: poly-commitment/benches/poly_comm.rs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+use poly_commitment::PolyComm;
+use rand_core::{CryptoRng, RngCore};
+
+fn helper_generate_random_elliptic_curve_point<RNG, P>(rng: &mut RNG) -> Affine<P>
+where
+    P::BaseField: PrimeField,
+    RNG: RngCore + CryptoRng,
+    P: SWCurveConfig,
+{
+    let p1_x = P::BaseField::rand(rng);
```
Comment (volhovm) [inline poly-commitment/benches/poly_comm.rs:17]: You could have just generated a scalar and scaled by it probably? I'm also thinking there's /likely/ a way to generate random points right away, no?

**Review by volhovm — APPROVED** [review]

File: poly-commitment/benches/poly_comm.rs (hunk 1 above)
Comment (dannywillems) [reply poly-commitment/benches/poly_comm.rs:17]: Huhu. YES!

Comment (dannywillems) [reply poly-commitment/benches/poly_comm.rs:17]: "Pourquoi faire simple quand on peut faire compliqué ?"

Comment (dannywillems) [reply poly-commitment/benches/poly_comm.rs:17]: See https://github.com/o1-labs/proof-systems/pull/2612

**Pull request merged**